sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/publishing/InfluxDBPublisher.java

```diff
@@ -19,14 +19,16 @@
 
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Objects.requireNonNull;
-import static org.apache.beam.repackaged.core.org.apache.commons.lang3.StringUtils.isNoneBlank;
 
 import java.io.IOException;
 import java.util.Collection;
+import java.util.Map;
+import org.apache.beam.repackaged.core.org.apache.commons.lang3.StringUtils;
```

Comment:
argh this commons import escaped here too. Can you use the non repackaged version please.

Reply:
ah..I would swear I changed it...thanks!